pkg/engine/template_generator.go

```diff
@@ -295,6 +295,18 @@ func getContainerServiceFuncMap(cs *api.ContainerService) template.FuncMap {
 			}
 			return kc.GetOrderedKubeletConfigStringForPowershell()
 		},
+		"HasKubeReservedCgroup": func() bool {
+			kc := cs.Properties.OrchestratorProfile.KubernetesConfig
```

Comment:
May need some tuning when really enable this in AKS due containerd in agentpool level